src/test/ui/panic-runtime/abort-link-to-unwind-msvc-no-std-link-to-msvcrt.rs

```diff
@@ -0,0 +1,32 @@
+// build-pass
+// compile-flags: -C panic=abort
+// aux-build:exit-success-if-unwind-msvc-no-std.rs
+// only-msvc
+// Test that `no_std` with `panic=abort` under MSVC toolchain
```

Comment:
Do we also expect this to run successfully (even though it doesn't currently)? Since MSCRT is linked in?